src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -853,7 +853,6 @@ def load_arguments(self, _):
             'source_resource_id',
             options_list=[
                 '--source-resource-id',
-                c.deprecate(target='-r', redirect='--source-resource-id', hide=True),
             ],
```

Comment:
As there's no other option form, and the default option is `--source-resource-id`, may omit this

Reply:
omitted